src/org/ods/orchestration/service/leva/ProjectDataBitbucketRepository.groovy

```diff
@@ -0,0 +1,49 @@
+package org.ods.orchestration.service.leva
+
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurperClassic
+import org.ods.util.IPipelineSteps
+
+import java.nio.file.NoSuchFileException
+
+
+class ProjectDataBitbucketRepository {
+
+    static final String BASE_DIR = 'projectData'
+
+    private IPipelineSteps steps
+
+    ProjectDataBitbucketRepository(IPipelineSteps steps) {
+        this.steps = steps
+    }
+
+    String save(Object jiraData, String version) {
+        def fileName = "${BASE_DIR}/${version}.json"
+        steps.writeFile(
+            file: fileName,
+            text: JsonOutput.prettyPrint(JsonOutput.toJson(jiraData))
+        )
+        return fileName
+    }
+
+    Object loadFile(String fileName) {
+        try {
+            def savedData =  this.steps.readFile(file: "${BASE_DIR}/${fileName}.json")
```

Comment:
not a big deal more a question of code style... there is no need to use this `this.`

Reply:
@stitakis we use `this` references in a larger portion of the Release Manager, which aids readability at least for me. `this` shows me that you access a member, vs. without `this` can only reference a method argument.